integration/deterministic-build-output-test.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from "@playwright/test";
+import globby from "globby";
+import fs from "fs";
+import path from "path";
+
+import { createFixtureProject, js } from "./helpers/create-fixture";
```

Comment:
'js' is defined but never used
```suggestion
import { createFixtureProject } from "./helpers/create-fixture";
```